examples/sapper/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sapper-example",
+  "scripts": {
+    "dev": "sapper dev",
+    "build": "sapper build --legacy",
+    "export": "sapper export --legacy",
+    "start": "node __sapper__/build"
+  },
+  "private": true,
+  "dependencies": {
+    "@tinyhttp/app": "workspace:*"
+  },
+  "devDependencies": {
+    "sapper": "^0.28.0",
```

Comment:
hm, why is sapper a dev dependency here, as well as svelte? should it be like this?

Reply:
This is what they do in the template. I think its because it is only used for building, and not for the program itself.

Reply:
@shzmr understood